resources/ory/charts/hydra/templates/service-admin.yaml

```diff
@@ -13,6 +13,10 @@ spec:
       targetPort: http-admin
       protocol: TCP
       name: http
+    - port: 15020
```

Comment:
AFAIR Hydra does not expose port 15020, what's the reason for this change?

Reply:
The istio sidecar exposes that port, and [Istio probe rewrite](https://istio.io/docs/ops/configuration/mesh/app-health-check/#probe-rewrite) is configured so the health endpoint for hydra looks like this:
`#> kubectl get pod -n kyma-system ory-hydra-c8879d574-tjvr8 -o yaml`
```
    livenessProbe:
      failureThreshold: 5
      httpGet:
        path: /app-health/hydra/livez
        port: 15020
        scheme: HTTP
```

Reply:
after offline discussion, this aims to provide accessibility for 3rd party service, that needs to query enpoints. From kubelet POV it is not required at all. I have verified that exposing pilot-agent port, therefore exposing `/quitquitquit` enpoint will not be an issue due to check for localhost requests only.